src/components/frame/v5/pages/VerifiedPage/partials/VerifiedTable/hooks.tsx

```diff
@@ -0,0 +1,106 @@
+import { ColumnDef, createColumnHelper } from '@tanstack/react-table';
+import React, { useMemo } from 'react';
+import { formatText } from '~utils/intl';
+import Icon from '~shared/Icon';
+import { ColonyContributorFragment } from '~gql';
+import MemberAvatar from '../MemberAvatar';
+import Checkbox from '~v5/common/Checkbox';
+import { ContributorTypeFilter } from '~v5/common/TableFiltering/types';
+import UserStatusComponent from '~v5/shared/CardWithBios/partials/UserStatus';
+import PermissionRow from '../PermissionRow';
+import { getEnumValueFromKey } from '~utils/getEnumValueFromKey';
+
+export const useVerifiedTableColumns = (
+  name: string,
+): ColumnDef<ColonyContributorFragment, string>[] => {
+  const columnHelper = useMemo(
+    () => createColumnHelper<ColonyContributorFragment>(),
+    [],
+  );
+  const onDeleteClick = () => {};
+
+  const columns: ColumnDef<ColonyContributorFragment, string>[] = useMemo(
+    () => [
+      columnHelper.display({
+        id: 'checkbox',
+        size: 20,
+        cell: ({ row }) => (
+          <Checkbox
+            id={`verified-${row.original.contributorAddress}`}
+            name={`verified-${row.original.contributorAddress}`}
```

Comment:
I think you can skip these and add default id inside using `useId` like we do in Switch src/components/v5/common/Fields/Switch/Switch.tsx